File: content/0-basics/2-hello-world/mastra/3-agent_gemini.ts
import { Agent } from "@mastra/core/agent";

// model specified as "provider/model-name" string
const LLM_MODEL = "google/gemini-pro-latest";

const agent = new Agent({
  name: "hello-world",
  instructions: "You are a helpful assistant.",
  model: LLM_MODEL,
});

const result = await agent.generate("What is the capital of France?");
console.log(result.text);
// "Paris."

File: content/0-basics/2-hello-world/pydantic/3-agent_gemini.py
from pydantic_ai import Agent

# model specified as "provider:model" string
LLM_MODEL = "google-gla:gemini-pro-latest"

agent = Agent(LLM_MODEL)

result = agent.run_sync("What is the capital of France?")
print(result.output)
# "Paris."
